src/tools/genpolicy/rules.rego

```diff
@@ -801,223 +809,243 @@ mount_source_allows(p_mount, i_mount, bundle_id, sandbox_id) {
 
 ######################################################################
 # Storages
-
 allow_storages(p_storages, i_storages, bundle_id, sandbox_id) {
-    p_count := count(p_storages)
-    i_count := count(i_storages)
-    print("allow_storages: p_count =", p_count, "i_count =", i_count)
```

Comment:
Printing the count can be useful for debugging.

Reply:
Fixed.